pages/events/index.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="pattern-bg">
+    <div class="content">
+      <section class="result">
+        <h2 class="section-title">Eventos</h2>
+        <Spinner v-if="loadingEvents" class="spinner" />
+        <div v-else>
+          <ul class="event-list">
+            <li v-for="event in Events" :key="event.id" class="event-card-item">
+              <EventCard :data="event" />
+            </li>
+          </ul>
+          <span
+            @click="toggleMenu"
+            v-show="showSeeMore"
+            class="button button-see-more"
+            >Ver todos os eventos</span
+          >
+        </div>
+      </section>
+    </div>
+  </div>
+</template>
+
+<script>
+import EventCard from '~/components/EventCard'
+import Spinner from '~/components/Spinner'
+
+export default {
+  components: {
+    EventCard,
+    Spinner,
+  },
+  data() {
+    return {
+      showSeeMore: true,
+      loadingEvents: false,
+      Events: [],
+    }
+  },
+  mounted() {
+    this.getEvents()
+  },
+  methods: {
+    async getEvents() {
+      this.loadingEvents = true
+      this.AllEvents = await this.$eventRepository.index()
+      this.Events = this.AllEvents.slice(0, 4)
```

Comment:
- Use lower camelcase: `this.events`
- Instead of slice can you use the api pagination?
- Why only 4 events?

Reply:
How to use api pagination? Is there a sample?